HHS/OPRE-OPS
# fix: fixes logic for displaying requestor name

fix: adds createdBy prop for budget lines

## What changed

Fixes logic for displaying requestor name of the Review Card on the Review page. Found bug during demo on 9/12/2024. 

## Issue

#2688 

## How to test

1. create a BLI on a Agreement
2. submit for status change
3. on review page, logged in user should display in the review card
4. 💰 profit

## Screenshots

https://github.com/user-attachments/assets/2598267d-f679-492d-b314-86e12e92d2c8

## Definition of Done Checklist
